aiohttp_retry/client.py

```diff
@@ -64,9 +64,8 @@ def _is_status_code_ok(self, code: int) -> bool:
     async def _do_request(self) -> ClientResponse:
         current_attempt = 0
         while True:
-            current_attempt += 1
-            self._logger.debug("Attempt {} out of {}".format(current_attempt, self._retry_options.attempts))
-            if current_attempt > 1:
+            self._logger.debug(f"Attempt {current_attempt} out of {self._retry_options.attempts}")
+            if current_attempt > 0:
                 retry_wait = self._retry_options.get_timeout(current_attempt)
                 await asyncio.sleep(retry_wait)
```

Comment:
Your solution doesn't increase current_attempt anymore
This is incorrect, because will lead to infinite loop

Reply:
indeed, fixed :slightly_smiling_face: